### PR TITLE
Carriers are assigned a position on connection.

### PR DESCRIPTION
**Pull request by LMiljak**
While this PR contains a lot of unfinished (but working) code, I noted that many of you needed this to identify a certain client. So instead of reviewing like you used to, just check that there aren't weird buggy stuff in the code.

You can get the position a client is assigned from the PositionHolder class, which kind of acts as a global variable.
The CarrierAssigner class in Desktop module assigns a newly connected client a position by sending it a PositionMessage.

### CODE REVIEW

**Comment by NilsHullegien** [comment]
Looks good to me and functions well. We will indeed have to clean up some stuff, but for now it looks good, I approve.

File: ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java [hunk 1]
```diff
@@ -28,8 +33,12 @@
     public Commander(Vector3f startLocation) {
         super();
         setModel(getDefaultModel());
+        Arrays.asList("a", "b");
```
Comment (marcelkuipers) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Commander.java]: This line appears to do nothing

File: ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Platform.java [hunk 2]
```diff
@@ -13,18 +19,62 @@
...
+    }
+    
+    /**
+     * Checks if the platform is fully being carried by four carriers.
+     * 
+     * @return
+     * 		True iff the platform is carried by four carriers.
+     */
+    public boolean isFull() {
+    	for (PlatformPosition position : PlatformPosition.values()) {
```
Comment (marcelkuipers) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/entity/Platform.java]: I think you could also use carriers.size()

**Comment by marcelkuipers** [comment]
Besides my minor comments, this PR looks good.

**Comment by marcelkuipers** [comment]
Looks good now! Merging

**Pull request merged**